src/MOI_wrapper.jl

```diff
@@ -431,5 +433,128 @@ end
     end
     # qmatind is ::Vector{Cint}, so we convert back to ::Vector{Int}.
     return sparse(Int.(qmatind), qmatcol, qmatval)
- end
+end
+
+"""
+    compute_conflict(model::Optimizer)
+
+Compute a minimal subset of the constraints and variables that keep the model
+infeasible.
+
+See also `CPLEX.ConflictStatus`, `CPLEX.VariableConflictStatus`, and
+`CPLEX.ConstraintConflictStatus`.
+
+Note that if `model` is modified after a call to `compute_conflict`, the
+conflict is not purged, and any calls to the above attributes will return values
+for the original conflict without a warning.
+"""
+function compute_conflict(model::Optimizer)
+    try 
+        model.conflict = c_api_getconflict(model.inner)
+    catch exc
+        if isa(exc, CplexError) && exc.code == CPXERR_NO_CONFLICT
+            model.conflict = ConflictRefinerData(CPX_STAT_CONFLICT_FEASIBLE, 0, Cint[], Cint[], 0, Cint[], Cint[])
+        else
+            throw(exc)
```

Comment:
Use `rethrow(exc)`.

Can you add a comment about why the `try-catch` is necessary?